Cargo.toml

```diff
@@ -101,6 +101,8 @@ backend-singlepass = [
 wasi = ["wasmer-wasi"]
 managed = ["backend-singlepass", "wasmer-runtime-core/managed"]
 
```

Comment:
Remove the empty line between the features please.

Reply:
@Hywan I wanted to stay with the feature flag name of `wasmparser`

Reply:
but it's your call of course

Reply:
@MarkMcCaskey @Hywan 
please LMK what's your decision

Reply:
@MarkMcCaskey @Hywan 
I've renamed the feature flag to `deterministic-execution`